notes/Chapter-Auctions/chap7.tex

```diff
@@ -64,7 +64,7 @@ \section{Definitions and Assumption}
 \item In the following we will always consider auctions where a single good is allocated to one of the bidders, hence $X=N$. An element of $X$ corresponds to the winner of the auction. Moreover, we assume the following (quasilinear) structure of the utility function of each player $i$: $u_i(x,p,v) = v_i \cdot \delta_{x = i} - p_i$, where $v_i$ is the value $i$ associates to winning the auction, $\delta_{x = i} = 1$ if $x = i$. Thus, the utility of player $i$ is $v_i - p_i$ if the item is allocated to her and she has to pay $p_i$, and just $-p_i$ if she has to make the payment $p_i$ without obtaining the item.
 
 The utility functions imply that each agent has the same value-for-money. Consequently, we have \emph{transferable utility}. Each agent can transfer any given amount of utility to another agent by giving her an appropriate sum of money (and the sum of the total utilities of all the players remains the same).
-Note further that agents with such utility functions are \emph{risk neutral}. Suppose that an agent must decide whether she wants to participate in a fair lottery, that awards the amount $x$ half of the time and the amount $-x$ the other half of the time. The expected utility is zero--- thus the agent is indifferent between participating in the lottery or not participating. In real live the situation is often more complicated, as for instance \emph{risk seeking} agents might prefer to participate in a lottery that cost \$\,$1$ and has a 0.001 \% chance of paying off \$\,$1000$, whereas a risk neutral player is indifferent.
+Note further that agents with such utility functions are \emph{risk neutral}. Suppose that an agent must decide whether she wants to participate in a fair lottery, that awards the amount $x$ half of the time and the amount $-x$ the other half of the time. The expected utility is zero--- thus the agent is indifferent between participating in the lottery or not participating. In real live the situation is often more complicated, as for instance \emph{risk seeking} agents might prefer to participate in a lottery that cost \$\,$1$ and has a 0.01 \% chance of paying off \$\,$1000$, whereas a risk neutral player is indifferent.
```

Comment:
Okay, I don't think I will have the time to master GitHub in a couple of hours, but it is fun to use it! Furthermore, I don't think I will contribute a lot between now and the exam since I will focus on the theory now :)
However, I have a new problem, this time with the interface GitHub Desktop: I have created a Pull Request here: https://github.com/gauthmug/LINMA2345/pull/1
Obviously, it shouldn't be there, and I am not even sure you can see what's there :'(